docs/meshstack.azure.index.md

```diff
@@ -395,3 +396,112 @@ The arguments available here are:
+                "field": "type"
+              },
+              {
+                "allOf": [
+                  {
+                    "field": "Microsoft.Authorization/roleAssignments/roleDefinitionId",
+                    "equals": "/subscriptions/<SUBSCRIPTION_ID>/providers/Microsoft.Authorization/roleDefinitions/<AZURE_SERVICE_PRINCIPAL_ROLE_ID>"
+                  },
+                  {
+                    "field": "Microsoft.Authorization/roleAssignments/principalId",
+                    "equals": "<AZURE_SERVICE_PRINCIPAL_ID>"
+                  },
+                ]
+              }
+            ]
+          },
+          "then": {
+            "effect": "audit"
+          }
+      }
+    }
+}
+```
+
+If you intend to call Azure functions via a Landing Zone configuration, then you need to grant the replicator principal also permissions to the role you have created for the [Azure secured function call](#secure-azure-function-setup).
+
+#### Azure Function Access Setup
+
+In order to make an Azure Function only accessible via the replicators Service Principal follow these steps:
+
+1. Create a system assigned identity for your function (this is only required if you need the function to have permissions for Azure based resources like starting VMs, connecting Log Workspaces etc).
+
+    ![System assigned identity](assets/azure_function/system-assigned-identity.png)
+
+2. Lock down your function to only allow assigned users.
+
+    ![Assigned users only](assets/azure_function/assigned-users.png)
+
+3. Create a custom [Application Role](https://docs.microsoft.com/en-us/azure/architecture/multitenant-identity/app-roles). Its only possible to assign real users and unfortunatly no Service Principals directly to the function so this additional steps are required. Edit the Application Roles manifest like in this JSON:
+
+    ```json
+    {
+      "allowedMemberTypes": [
+        "Application"
+      ],
+      "description": "Allows an SPP to get a token to a restricted application",
+      "displayName": "SPP-Access",
+      "id": "<RANDOM_UUID>",
+      "isEnabled": true,
+      "lang": null,
+      "origin": "Application",
+      "value": "Access"
+    }
+    ```
+
+    ![App Role Manifest](assets/azure_function/app-role-manifest.png)
+
+4. In your Service Principals API permissions screen, add the newly created Application Role. Don't forget to grant admin consent again afterwards.
+
+    ![Assign the Application Role to SP](assets/azure_function/sp-token.png)
+
+
+After these steps, you should be able to fetch a token scoped to this Application Role (via the Service Principal secrets). The scope field inside the JWT token should match the application's ID. With this token you can call the Azure Function. This is the same functionality that also the meshReplicator is using.
+
+![Fetch Token](assets/azure_function/fetch-token.png)
+
+![Decoded Token](assets/azure_function/decoded-token.png)
+
+#### meshFed Identity Lookup
```

Comment:
d: this belongs into Identity Lookup configuration page, not here where we set up meshPlatform